.gitattributes

```diff
@@ -0,0 +1,10 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
Whats this file about?

Reply:
gitattributes allows you to tell git that files should be treated in a given way. You can use git attributes to apply attributes, one instance would be `diff`. For e.g., `*.rb diff=ruby` tells git that files ending in *.rb should be treated as Ruby code during diff operations.